main.nf

```diff
@@ -1490,7 +1493,49 @@ process SPADES {
 }
 
 /*
- * STEP 6.3.1: Run Blast on SPAdes de novo assembly
+ * STEP 6.3.1: Overlap scaffolds with Minimap2, induce and polish assembly, and call variants with seqwish and vg
+ */
+// TODO nf-core: What would the value of $PREFIX by for a multi-fasta?

```

Comment:
By this you mean a viral reference genome (in fasta format) that contains more than one sequence?  I am not sure exactly how the ` -p \$PREFIX` works, will look into that

Reply:
Yep. Exactly :+1: It's working right now because there is a single sequence in the fasts.